Jenkinsfile

```diff
@@ -61,6 +61,12 @@ pipeline {
             }
         }
 
+        stage('Run Lint') {
+            steps {
+                sh 'docker-compose -f docker-compose-lint.yaml up'
```

Comment:
@rranjan3 I assume that you have tested the negative scenario which CI system captures and fails.

Reply:
@pankajgoyal2 Good point. @rranjan3 Could you confirm that CI build will fail in case of lint error

Reply:
The service(within docker-compose) does return an exit code 1 in case of failures. Should that suffice or need to add extra handling?

Reply:
I suggest, please introduce some lint specific vulnerability intentionally and run it...it should exit without even going to Build stage.

Reply:
Updated @pankajgoyal2 to handle that. There's a flag specifically to pass out the return status outside docker-compose - https://docs.docker.com/compose/reference/up/